apps/sms/views/inbox/js/inbox.js

```diff
@@ -39,6 +39,11 @@ var InboxView = {
   // Set to |true| when in edit mode
   inEditMode: false,
 
+  /**
+   * Indicates whether user should be notified about newly saved draft.
+   */
+  notifyAboutSavedDraft: false,
```

Comment:
but does it work in split view ? Because onDraftSaved will not be called in that case, unless I miss something ?